hooks/ceph_hooks.py

```diff
@@ -637,6 +640,8 @@ def notify_client():
             service_name = get_client_application_name(relid, unit)
             if service_name:
                 data.update({'key': ceph.get_named_key(service_name)})
+                if "prometheus-ceph-exporter" in service_name:
+                    data.update({"version": ceph.resolve_ceph_version(hookenv.config("source") or "distro")})
```

Comment:
Why not add this attribute to all the `ceph-client` relations? For now, it seems that only the Ceph exporter would use it, but the application name can be any that the user may want.